lib/crapto/examples/repeating_key_xor_solve.nit

```diff
@@ -0,0 +1,24 @@
+import base64
+import crapto
+
+# Read the cipher from the file
+var reader = new FileReader.open("repeating_key_xor_cipher.txt")
+var cipher = reader.read_all
+var cipher_bytes: Bytes
+
+# Decode the base64 encoded ciphertext
+cipher = cipher.decode_base64
+cipher_bytes = cipher.to_bytes
```

Comment:
It is shameful that `decode_base64` returns a String. It should be a Bytes.

However, there is also a `decode_base64` in Bytes that may do the right things.

    var cipher_bytes = "repeating_key_xor_cipher.txt".to_path.read_all_bytes.decode_base64